generator/generator_config.textproto

```diff
@@ -482,6 +482,71 @@ service {
   retryable_status_codes: ["kUnavailable"]
 }
 
+
+# BigQueryControl
+service {
+  service_proto_path: "google/cloud/bigquery/v2/dataset.proto"
+  product_path: "google/cloud/bigquerycontrol/dataset/v2"
```

Comment:
For all of these, why not `google/cloud/bigquerycontrol/v2` ?

It seems simpler and less verbose to me.